packages/grafana-e2e-selectors/src/selectors/pages.ts

```diff
@@ -183,7 +183,7 @@ export const Pages = {
       container: 'data-testid Explore',
       graph: 'Explore Graph',
       table: 'Explore Table',
-      scrollBar: () => '.scrollbar-view',
+      scrollView: 'data-testid explorer scroll view',
```

Comment:
Not 100% sure what's going on here. Does the new nav introduce a custom scrollbar and now we need to grab a specific scroll view for the test?

I'm guessing the likelihood of `e2e.pages.Explore.General.scrollBar()` being used outside of core is unlikely as explore doesn't make use of plugins, right? It's worth noting that changing these selector keys can be problematic when running plugin integration tests across multiple versions of Grafana. Perhaps we keep the key as is and only change the value? 🤔

Reply:
The actual Explore page did not use this selector, and it was only returning a class name that was used internally within the `react-custom-scrollbars-2` component. No where did we actually pass this class name into the component, and there was no guarantee this selector was actually correct (apart from tests failing, which is fair)

This selector matched _all_ scroll views on the page, not just the Explore one. This PR adds a second scroll view onto the page, and broke this test because it was not correctly selector the Explore page scroller.